ftw/linkchecker/tests/test_functionality.py

```diff
@@ -22,19 +24,21 @@
 
 class TestLinkChecker(FunctionalTestCase):
 
+    def setUp(self):
+        super(TestLinkChecker, self).setUp()
+        self.portal2 = aq_parent(self.portal).get(SECOND_PLONE_SITE_ID)
```

Comment:
You may move this to the `LinkcheckerLayer`

Reply:
👍